modules/apps/transfer/keeper/relay_test.go

```diff
@@ -885,7 +885,7 @@ func (suite *KeeperTestSuite) TestOnAcknowledgementPacketSetsTotalEscrowAmountFo
 func (suite *KeeperTestSuite) TestOnTimeoutPacket() {
 	var (
 		path            *ibctesting.Path
-		amount          sdkmath.Int
+		amount          string
```

Comment:
I changed this test to take string directly so that we can also test the case of a malformed string amount, which was not currently covered. I discovered this when going through the coverage report related to the original issue. I can also pull this out into a separate PR.